app/javascript/components/charts/bubble/MobileChart.js

```diff
@@ -0,0 +1,125 @@
+import React, {useState} from 'react';
+import Select, { components } from 'react-select';
+import PropTypes from 'prop-types';
+import chevronDownIconBlack from '../../../../assets/images/icon_chevron_dark/chevron_down_black-1.svg';
+import chevronUpIconBlack from '../../../../assets/images/icon_chevron_dark/chevron-up.svg';
+
+
+const LEVELS_SUBTITLES = {
+  0: 'Unaware',
+  1: 'Awareness',
+  2: 'Building capacity',
+  3: 'Integrating into operational decision making',
+  4: 'Strategic assessment'
+};
+
+const blueDarkColor = '#2E3152';
+const secondaryColor = '#828397';
+
+const customSelectTheme = theme => ({
+  ...theme,
+  colors: {
+    ...theme.colors,
+    primary50: 'inherit'
+  }
+});
+
+const customSelectStyles = {
+  option: (provided, state) => ({
+    ...provided,
+    color: state.isSelected ? blueDarkColor : secondaryColor,
+    backgroundColor: state.isSelected ? 'inherit' : 'inherit',
+    fontWeight: state.isSelected ? '800' : 'normal'
+  }),
+  control: (provided) => ({
+    ...provided,
+    border: '1px solid',
+    boxShadow: '1px solid #2E3152',
+    borderRadius: 0
+  }),
+  indicatorSeparator: () => ({
+    display: 'none'
+  }),
+  menu: (provided) => ({
+    ...provided,
+    margin: 0,
+    borderRadius: 0
+  }),
+  container: (provided) => ({
+    ...provided,
+    margin: '0 0.75rem 1.2rem'
+  })
+};
+
+const DropdownIndicator = (props) => (
+  <components.DropdownIndicator {...props}>
+    <img src={props.selectProps.menuIsOpen ? chevronUpIconBlack : chevronDownIconBlack} />
+  </components.DropdownIndicator>
+);
+
+
+const MobileBubbleChart = ({ levels }) => {
```

Comment:
Hmm, I just realized, that this is not a chart but more like CompaniesAccordion or MobileSectorsOverview duno. Maybe it is better to rename that component?